src/lib/mina_base/test/mina_base_test.ml

```diff
@@ -0,0 +1,115 @@
+open Alcotest
+open Mina_base
+
+let () =
+  run "Test mina_base."
+    [ Zkapp_account_test.
+        ( "zkapp-accounts"
+        , [ test_case "Events pop after push is idempotent." `Quick
+              (checked_pop_reverses_push (module Zkapp_account.Events))
+          ; test_case "Actions pop after push is idempotent." `Quick
+              (checked_pop_reverses_push (module Zkapp_account.Actions))
+          ] )
+    ; Account_test.
+        ( "accounts"
+        , [ test_case "Test fine-tuning of the account generation." `Quick
```

Comment:
Very minor: test titles shouldn't have "."  

The current change actually preservers the old names, which I think makes sense.
 If you agree, we can make the removal of dots in test titles a further issue.